contrib/mesos/pkg/scheduler/podtask/pod_task_test.go

```diff
@@ -54,21 +131,27 @@ func TestEmptyOffer(t *testing.T) {
 		}},
 	}
 
-	defaultPredicate := NewDefaultPredicate(mresource.DefaultDefaultContainerCPULimit, mresource.DefaultDefaultContainerMemLimit)
-	if ok := defaultPredicate(task, nil, nil); ok {
-		t.Fatalf("accepted nil offer")
-	}
-	if ok := defaultPredicate(task, &mesos.Offer{}, nil); ok {
+	defaultProc := NewDefaultProcurement(
+		&mesos.ExecutorInfo{
+			Command: &mesos.CommandInfo{
+				Arguments: []string{},
```

Comment:
yes, please do. we don't want nasty surprises later, and it will make these test cases slightly easier on the eyes